src/workloads/execution/FilterWithLogicalExpressions.yml

```diff
@@ -0,0 +1,162 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  This workload stresses the query execution engine by running queries with complex logical
+  expressions that never match a document in the collection.
+
```

Comment:
Did you fill the form for [new workload requests](https://wiki.corp.mongodb.com/display/STAR/New+Workload+Requests)?

Reply:
Done

Reply:
Thanks